src/patternfly/components/ModalBox/docs/code.md

```diff
@@ -10,13 +10,16 @@ A modal box is a generic rectangular container that can be used to build modals.
 | `aria-labeledby="TitleID"` | `.pf-c-modal-box` | Gives the modal an accessible name by referring to the element that provides the dialog title. **Required** |
```

Comment:
Here's a suggestion on how to clarify `aria-label` vs `aria-labelledby`, related to my previous comment:

| Attribute | Applies to | Outcome |
| -- | -- | -- |
| `aria-labeledby="[id value of .pf-c-modal-box__header-title"` | `.pf-c-modal-box` | Gives the modal an accessible name by referring to the element that provides the dialog title. **Required when .pf-c-modal-box__header-title is present** |
| `aria-label="[title of modal]"` | `.pf-c-modal-box` | Gives the modal an accessible name. **Required when .pf-c-modal-box__header-title is _not_ present** |